front/src/components/FeedFAB.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import { ReactComponent as ZoomBtn } from '../assets/icons/zoom_out.svg';
+import { ReactComponent as WriteBtn } from '../assets/icons/add_circle.svg';
+
+const FloatingContainer = styled.div<{ ref: any }>`
```

Comment:
엘리먼트의 타입은 잘 모르겠지만 23라인을 보면 아래처럼 `any`대신 아래처럼 바꿔도 되지 않을까요?

```suggestion
const FloatingContainer = styled.div<null | HTMLElement>`
```

Reply:
실험해봤는데 안되네요.. ref의 타입이 뭔지 모르겠어요 ㅠ